cmake/modules/AddSwift.cmake

```diff
@@ -1833,7 +1840,21 @@ function(_add_swift_executable_single name)
       BINARY_DIR ${SWIFT_RUNTIME_OUTPUT_INTDIR}
       LIBRARY_DIR ${SWIFT_LIBRARY_OUTPUT_INTDIR})
 
+  # On Windows, libraries are prefixed with "lib", so we need to adjust FAT_LIBRARIES for this.
```

Comment:
Odd: every single library built at the end of the swift build process has the prefix `lib`

That said, LLVM's built libraries don't have the `lib` prefix, so perhaps I just haven't ifdefed the place we add `lib` as a prefix to be non-Windows